packages/app-desktop/gui/MainScreen/MainScreen.tsx

```diff
@@ -684,6 +692,12 @@ class MainScreenComponent extends React.Component<Props, State> {
 			);
 		} else if (this.props.mustUpgradeAppMessage) {
 			msg = this.renderNotificationMessage(this.props.mustUpgradeAppMessage);
+		} else if (this.props.showInvalidJoplinCloudCredential) {
+			msg = this.renderNotificationMessage(
+				_('Your Joplin Cloud credentials are invalid., please re-authenticate.'),
```

Comment:
```suggestion
				_('Your Joplin Cloud credentials are invalid, please login.'),
```

And please also update the other string